### DISCUSSION TITLE
Quicksearch Function Return an empty response?

### DISCUSSION THREAD

**Discussion by arulselvan008** [pimcore/pimcore#15355] [Q&A / Support] [answered]
Hello team

I have the Pimcore enterprise edition (v10.5.21) installed, but I am unable to access the Quick Search feature.

receiving only an empty response. If any extra features need to be configured or bundles added, please let me know.

Quick search :Asset
![image](https://github.com/pimcore/pimcore/assets/106297450/1824df9f-9da8-433f-88a3-02492b817e67)

Quick search :DataObject
![image](https://github.com/pimcore/pimcore/assets/106297450/65c2d286-29da-4278-baf9-5972a284cbac)

Regards,
Arul

**Comment by kingjia90** [comment, marked as answer]
Hi Arul,

Could you please run `bin/console pimcore:search-backend-reindex` and retry? 
Let us know if it works. 

Best regards,
JJ

**Reply by arulselvan008** [reply]
Thanks JJ,

It's working well.
![image](https://github.com/pimcore/pimcore/assets/106297450/9857dd9c-2e4c-4a79-9700-941b477ce83e)

**Comment by arulselvan008** [comment]
I have one more doubt. 
How do I set the page preconditions for the portal engine? The roles "Admin" and "Editor" were both created.
Is it possible to build through a new configuration off of it, or is it possible to do the backend coding only?

**Reply by kingjia90** [reply]
For this, it's better to write to the enterprise support.
Not sure if i got it right, but this may be helpful for the issue you mention https://github.com/pimcore/portal-engine/blob/3.x/doc/15_Development_Documentation/15_Customize_and_Extend_Behavior/11_PreCondition_Service.md
